priv/static/specs/http_api_spec.yaml

```diff
@@ -79,3 +134,59 @@ definitions:
     properties:
       version:
         type: "string"
+  SessionsList:
+    type: "object"
+    properties:
+      TotalCount:
+        type: integer
+      Sessions:
+        type: array
+        items:
+          $ref: '#/definitions/SessionInfo'
+  SessionInfo:
+    type: "object"
+    properties:
+      IMSI:
+        type: string
+      IMEI:
+        type: string
+      MSIDN:
+        type: string
+      Version:
+        type: string
+      RemoteControlIp:
+        type: string
+      RemoteDataIp:
+        type: string
+      ControlPort:
+        properties:
+          Name:
+            type: string
+          Ip:
+            type: string
+          Type:
+            type: string
+      StartTime:
```

Comment:
Some more information should also be added:

* QoS parameter
* PDN Type
* RAT Type
* UE IPs
* UE Time Zone
* AMBR
* APN
* User Location Information
* Serving Network

Some of those information have different encodings in GTPv1 and GTPv2